baseframe/__init__.py

```diff
@@ -27,13 +28,21 @@
 networkbar_cache = Cache(with_jinja2_ext=False)
 asset_cache = Cache(with_jinja2_ext=False)
 cache = Cache()
+dogpile = DogpileCache()
 babel = Babel()
 csrf = CSRFProtect()
 if DebugToolbarExtension is not None:
     toolbar = DebugToolbarExtension()
 else:
     toolbar = None
 
+
+DEFAULT_DOGPILE_CONFIG = {
+    'DOGPILE_CACHE_BACKEND': 'dogpile.cache.redis',
+    'DOGPILE_CACHE_URLS': 'http://127.0.0.1:6379',
```

Comment:
According to the [official code sample](https://bitbucket.org/artem_serga/flask-dogpile-cache), this should be `['127.0.0.1:6379']`.